radxa-e25: blue light on when entering loader mode

RadxaMitchell opened the pull request:
Setting the state of the lamp via the gpio command When the recovery button is pressed to enter loader mode, the rgb led displays blue color.the blue light should be always on.

The number of the gpio pin that controls the rgb_blue lamp is 149 --> GPIO4_C5
The number of the gpio pin that controls the rgb_green lamp is 17 --> GPIO0_C1

RadxaYuntian reviewed (commented):
U-Boot有LED命令，可以考虑和内核同步设备树后用LED命令来控制
另外#88 #89也应该是在同一个PR里面吧

RadxaMitchell commented:
#88 是和 #87 一起的，我分开提了
#89 是把波特率从 1500000 改为 115200 的

RadxaMitchell committed:
radxa-e25: blue light on when entering loader mode

When the recovery button is pressed to enter the loader mode,
execute the led command to light up rgb_blue.

Signed-off-by: Mitchell Ma <machuang@radxa.com>

RadxaYuntian reviewed (commented):
没看到dts添加led的提交

RadxaMitchell commented:
dts 我在 #88 这个提在了这个 pr 里面，还没提交正在提

RadxaYuntian commented:
所以这两个就应该是在一个PR里面

RadxaMitchell commented:
那我把 #88 关掉，从新在这 pr 里面提交下

RadxaMitchell committed:
dts: radxa-e25: add blue led node

Signed-off-by: Mitchell Ma <machuang@radxa.com>

RadxaYuntian commented on arch/arm/dts/rk3568-radxa-e25.dts:
```diff
@@ -37,4 +37,21 @@
+			u-boot,dm-pre-reloc;
+			label = "blue";
+			gpios = <&gpio4 RK_PC5 GPIO_ACTIVE_HIGH>;
+			default-state = "off";
```
```suggestion
```
只要我们用到`default-state`一般都是错误的用法：https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/common.yaml

RadxaStephen commented:
Added to branch next-dev-v2024.03.
